lib/WebService/Async/SmartyStreets.pm

```diff
@@ -0,0 +1,199 @@
+package WebService::Async::SmartyStreets;
+# ABSTRACT: Access SmartyStreet API
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+
+=HEAD
+
+WebService::Async::SmartyStreets - calls the SmartyStreets API and checks for the validity of the address
+
+=head1 SYNOPSIS
+    
+    my $ss;
+
+    $ss = WebService::Async::SmartyStreets->new(
+        auth_id => #insert auth_id,
+        token   => #insert token,
+        );
+    IO::Async::Loop->new->add($ss);
+
+    $ss = get_smartystreets();
+
+    my $addr = $ss->verify_international(<hash of address element>, geocode => 'true')->get;
+    print($addr->status);
+    
+=head1 DESCRIPTION
+
+This class calls the SmartyStreets API and parse the response to L<WebService::Async::SmartyStreets::Address>
+
+Note that this module uses L<Future::AsyncAwait>
+
+=over 4
+
+=cut
+
+use parent qw(IO::Async::Notifier);
+
+use mro;
+no indirect;
+
+use URI;
+use URI::QueryParam;
+
+use Future::AsyncAwait;
+use Net::Async::HTTP;
+use JSON::MaybeUTF8 qw(:v1);
+
+use WebService::Async::SmartyStreets::Address;
+
+use Log::Any qw($log);
+
+=head2 configure
+
+Configures the class with the auth_id and token
+Takes in: Hash which consists of auth_id and token
```

Comment:
This is misleading and the formatting makes it hard to see the full list. Please see our official documentation for this.

```
Takes the following named parameters:

=over 4

=item * C<auth_id> - ...

=item * C<token> - ...

=back

```